rasa/core/agent.py

```diff
@@ -863,7 +863,7 @@ def _create_domain(domain: Union[Domain, Text, None]) -> Domain:
 
         if isinstance(domain, str):
             domain = Domain.load(domain)
-            domain.check_missing_responses()
```

Comment:
This was a bug not noticed before, API v3 used in v2 🤔  Wonder when it would be run into.

Reply:
Actually I'm totally wrong 🤦  let me fix this.